docs/translations.md

```diff
@@ -24,12 +24,30 @@ exactly the languages that are included in the app.
    2. Add the language code to `language_values` in `app/core/src/main/res/values/arrays_general_settings_values.xml`
       so that the index in the list matches that of the newly added entry in `language_entries`.
 
+## Adding a component on Weblate
+
+When adding a new code module that is including translatable strings, a new components needs to be added to Weblate.
+
+1. Go the the Weblate page to [add a component](https://hosted.weblate.org/create/component/?project=3696).
+2. Switch to the "From existing component" tab.
+3. Enter a name for the component.
+4. For "Component", select "K-9 Mail/Thunderbird/ui-legacy".
+5. Press the "Continue" button.
+6. Under "Choose translation files to import", select "Specify configuration manually".
```

Comment:
In case the code path already exists, why not selecting it and import existing strings.

Reply:
In the past the list didn't always include the location I wanted to import. I figured the instructions would be easier to follow without branches (pick the location from the list if available, otherwise specify configuration manually).